sdk/core/token_library/token_unseal.S

```diff
@@ -127,8 +115,12 @@ _Z16token_obj_unsealP10SKeyStructP10SObjStruct:
 	.hidden  _Z23token_obj_unseal_staticP10SKeyStructP10SObjStruct
 	.globl   _Z23token_obj_unseal_staticP10SKeyStructP10SObjStruct
 _Z23token_obj_unseal_staticP10SKeyStructP10SObjStruct:
-	li       a2, CheriSealTypeStaticToken
-	j        .Ltoken_unseal_internal
+	auipcc     ca2, %cheriot_compartment_hi(__sealingkey)
+	clc        ca2, %cheriot_compartment_lo_i(_Z23token_obj_unseal_staticP10SKeyStructP10SObjStruct)(ca2)
+	li         t0, CheriSealTypeStaticToken
+    csetaddr   ca2, ca2, t0
+    csetbounds ca2, ca2, 1
```

Comment:
I guess we could avoid these extra instructions by having three versions of `__sealingKey` in memory and loading the appropriate one? I think that would be a net win on space and cycles?

Reply:
Yeah, I had just the same realization this morning: a `csetaddr; csetbounds` pair alone is the size of a capability.